docs/data/whats-new/WhatsNewLayout.tsx

```diff
@@ -0,0 +1,251 @@
+    announcementDate: 'Monday, Aug 12, 2022',
+    url: 'https://mui.com/blog/aggregation-functions/',
+    highlightList: [
+      {
+        title: 'Aggregation in summary rows and row groups',
+        url: 'https://mui.com/blog/aggregation-functions/#wait-what-is-an-aggregation-function',
+      },
+      {
+        title: 'Row pinning',
+        url: 'https://mui.com/blog/aggregation-functions/#row-pinning',
+      },
+    ],
+  },
+  {
+    title: 'New Premium plan',
+    description:
+      'Premium plan announcement, including the most advanced features for data analysis and management.',
+    announcementDate: 'Thursday, May 12, 2022',
+    url: 'https://mui.com/blog/premium-plan-release/',
+    highlightList: [
+      { title: 'Row Grouping', url: '/x/react-data-grid/row-grouping/' },
+      { title: 'Excel export', url: '/x/react-data-grid/export/#exported-rows' },
+    ],
+  },
+  {
+    title: 'MUI X v5.0.0',
+    description: 'A new virtualization engine, and improvements in several APIs.',
+    announcementDate: 'Monday, Nov 22, 2021',
+    url: 'https://mui.com/blog/mui-x-v5/',
+    highlightList: [
+      {
+        title: 'New virtualization engine',
+        url: 'https://mui.com/blog/mui-x-v5/#a-new-virtualization-engine',
+      },
+      {
+        title: 'Improved state management',
+        url: 'https://mui.com/blog/mui-x-v5/#improved-state-management',
+      },
+      {
+        title: 'Simplified style customization',
+        url: 'https://mui.com/blog/mui-x-v5/#simplified-style-customization',
+      },
+    ],
+  },
+];
+
+function BlogCard(props: { blog: Blog }) {
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        px: 2,
+        pt: 2,
+        pb: 0,
+        gap: 1.5,
+        borderRadius: '12px',
+        backgroundColor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? 'primaryDark.700'
+            : `${alpha(theme.palette.grey[50], 0.4)}`,
+        borderColor: 'divider',
+      }}
+      component="article"
+      variant="outlined"
+    >
+      <CardContent sx={{ p: 0 }}>
+        <Box
+          sx={{
+            borderBottom: '1px solid',
+            borderColor: 'divider',
+            marginBottom: 1,
+            px: 1,
+          }}
+        >
+          <Typography variant="caption" color="text.secondary">
+            {props.blog.announcementDate}
+          </Typography>
+          <Typography gutterBottom variant="h6" component="div">
+            {props.blog.title}
+          </Typography>
+        </Box>
+
+        <Box sx={{ p: 1 }}>
+          <Typography variant="body2" component="div" color="text.secondary">
+            {props.blog.description}
+          </Typography>
+
+          <List sx={{ pb: 0 }}>
+            {props.blog.highlightList.map((item) => (
+              <ListItem
+                sx={{
```

Comment:
You miss a `key` prop here to avoid error message in console

```suggestion
              <ListItem
                key={item.title}
                sx={{
```

You can also use the index since this list will not be modified